content/docs/addons-shallow-renderer.md

```diff
@@ -1,68 +1,69 @@
 ---
 id: shallow-renderer
-title: Shallow Renderer
+title: Renderização superficial
 permalink: docs/shallow-renderer.html
 layout: docs
 category: Reference
 ---
 
-**Importing**
+**Importando**
 
 ```javascript
 import ShallowRenderer from 'react-test-renderer/shallow'; // ES6
-var ShallowRenderer = require('react-test-renderer/shallow'); // ES5 with npm
+var ShallowRenderer = require('react-test-renderer/shallow'); // ES5 com npm
 ```
 
-## Overview {#overview}
+## Visão geral {#overview}
```

Comment:
```suggestion
## Visão geral {#visao-geral}
```

Reply:
thanks @jhonmike we decided to not translate links anymore. I created https://github.com/reactjs/pt-BR.reactjs.org/issues/42 in case you want to help